app/components/candidate_interface/complete_section_component.html.erb

```diff
@@ -1,8 +1,12 @@
-<%= form_with model: @application_form, url: @path, method: @request_method do |f| %>
-  <div class='govuk-form-group'>
-    <%= f.hidden_field @field_name, value: false %>
-    <%= f.govuk_check_box @field_name, true, multiple: false, label: { text: t('application_form.completed_checkbox') } %>
-  </div>
+<%= form_with model: completion_form, url: path, method: request_method do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <%= render(summary) %>
+
+  <%= f.govuk_radio_buttons_fieldset field_name, legend: { text: 'Have you completed this section?', size: 'm' } do %>
```

Comment:
@davidgisbey rightly pointed out that the CompleteForm can be simplified further if we use a standardised field (eg `:complete`)